prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div>Click here to go to the home page</div>
```

Comment:
Hi Frank, this page should include a link that takes you back to the home page (`../index.html`)

Reply:
okay thank you !